src/components/common/helpers/Helpers.tsx

```diff
@@ -189,7 +189,7 @@ export function getRandomColor(email: string): string {
 export function showError(serverError, showToastOnUnknownError = true, hideAccessError = false) {
     if (serverError instanceof ServerErrors && Array.isArray(serverError.errors)) {
         serverError.errors.map(({ userMessage, internalMessage }) => {
-            if (serverError.code === 403 && userMessage === 'unauthorized') {
+            if (serverError.code === 403 && (userMessage === 'unauthorized'|| 'forbidden')) {
```

Comment:
Please move unauthorized const and also move 'forbidden' to const

Reply:
done